frontend/src/app/main/ui/ds/foundations/heading.stories.jsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+import Components from "@target/components";
+
+const { Heading } = Components;
+const { StoryWrapper } = Components.storybook;
+
+export default {
+  title: "Foundations/Heading",
+  component: Components.Heading,
+};
+
+
+export const Display = {
+  render: () => (
+    <StoryWrapper theme="default" >
+      <Heading level="1" typography="display">This is a level 1 heading</Heading>
+    </StoryWrapper>
+  ),
+};
+
+export const Level1TitleLarge = {
+  render: () => (
+    <StoryWrapper theme="default">
+      <Heading level="1" typography="title-large">
+        This is a level 1 heading with Title-large typography
+      </Heading>
+    </StoryWrapper>
+  ),
+};
+
+ 
+export const Level1TitleMedium = {
+  render: () => (
+    <StoryWrapper theme="default">
+      <Heading level="1" typography="title-medium">
+        This is a level 1 heading with Title-medium typography
+      </Heading>
+    </StoryWrapper>
+  ),
+};
+
+export const Level1Code = {
+  render: () => (
+    <StoryWrapper theme="default">
+      <Heading level="1" typography="code-font">
+        This is a level 1 heading with code-font typography
+      </Heading>
+    </StoryWrapper>
+  ),
+};
+
+export const Level1HeadingLarge = {
+  render: () => (
+    <StoryWrapper theme="default">
+      <Heading level="1" typography="heading-large">
+        This is a level 1 heading with heading-large typography
+      </Heading>
+    </StoryWrapper>
+  ),
+};
+
+export const Level1HeadingMedium = {
+  render: () => (
+    <StoryWrapper theme="default">
+      <Heading level="1" typography="heading-medium">
+        This is a level 1 heading with heading-medium typography
+      </Heading>
+    </StoryWrapper>
+  ),
+};
```

Comment:
Our level has two props. Here, we should have two stories, one showing off each prop.

(later when/if we introduce the Controls addon, we could just have a single story and the user would be able to change themselves the prop values an experiment).